packages/patrol/lib/src/native/native_automator.dart

```diff
@@ -73,6 +74,22 @@ class NativeAutomatorConfig {
   /// Time to wait for native views to appear.
   final Duration findTimeout;
 
+  /// Whether to show the keyboard when entering text.
+  ///
+  /// The default is to always show the keyboard when entering text, and only
+  /// dismiss it after tapping on another UI element. However, that "another UI
+  /// element" might be obscured by the keyboard, which opens up an uncanny
+  /// valley of non-deteministic and hard-to-debug behavior.
+  ///
+  /// That's why you might want to set this to `false`, which:
+  ///  * on Android: will disable showing the keyboard at all
```

Comment:
Could Android's behavior be the same as the iOS's one? I think the iOS's one is better resembling the human actions, possibly invoking some layout problems/exceptions for when keyboard shows up

Reply:
No reliable methods to hide the keyboard during UI tests exist on Android. `pressBack` is the best we have.